monero-outreach-docs/translations/es/PSA/files/wallet/wallet_es.sla

```diff
@@ -60,19 +64,98 @@
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" FSHADE="90" CH="Tu monedero de Monero (XMR) es la caja fuerte que contiene tus XMR y los mantiene seguros. Si quieres asegurarte de mantenerlo así, existen buenas prácticas que quisieras seguir para mantener tus monederos lo "/>
+            <para/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" FSHADE="90" CH="más seguros posible."/>
+            <para/>
+            <para/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" FSHADE="90" CH="Los monederos ‘calientes’ son monederos que se conectan "/>
+            <para/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" FSHADE="90" CH="al internet, mientras que los monederos en ‘frío’ no. Los monederos en frío generalmente son más seguros pero "/>
+            <para/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" FSHADE="90" CH="vienen con un sacrificio a conveniencia del quien los usa."/>
+            <para/>
+            <para/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" FSHADE="90" CH="Un monedero físico, como los ya reconocidos populares Ledger o Trezor, pueden reconocerse como monederos intermedios entre los monederos ‘calientes’ y ‘fríos’ porque puedes conectarlos para enviar y/o recibir a un dispositivo conectado al internet, pero al desconectarlos no son "/>
+            <para/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" FSHADE="90" CH="accesibles a través del internet."/>
+            <para/>
+            <para/>
+            <ITEXT FONT="Adobe Caslon Pro Bold" FONTSIZE="12" CH="He aquí algunas prácticas en general para tener en "/>
+            <para LINESP="13.8"/>
+            <ITEXT FONT="Adobe Caslon Pro Bold" FONTSIZE="12" CH="cuenta cuando se trate de la seguridad del monedero:"/>
+            <para LINESP="13.8"/>
+            <para LINESP="13.8"/>
+            <ITEXT CH="1. Para evitar estafas que roben tus XMR, utiliza solamente monederos aprobados por la comunidad de Monero. "/>
+            <para LINESP="13.8"/>
+            <ITEXT CH="Investiga antes de configurar cualquier monedero. Un buen lugar para iniciar es la sección de información en el "/>
```

Comment:
"Un buen lugar para iniciar es la sección de información **en** ". Just "en" will do to talk about the subreddit. Alternatively, you can add the word "subreddit" in the next ITEXT line to make more sense.

Reply:
done